frontend/src/components/app/RawModal.js

```diff
@@ -232,8 +232,7 @@ class RawModal extends Component {
       }
 
       await this.removeModal();
-
-      await deleteView(windowType, viewId, type);
+      await deleteViewRequest(windowType, viewId, type);
```

Comment:
also, where possible and time/budget fit, pls rename `windowType` to `windowId`. So we would share the same ubiquitous across the whole application.

Reply:
We're over the budget already so I don't want to get sidetracked doing things that are not in the scope of this issue. I've only renamed this function because it has the same name as a redux action creator.

Reply:
ok, i understand.
Next time, pls make sure to consider the 'ubiquitous language` fixes when doing the estimation.